src/_igraph/graphobject.c

```diff
@@ -15997,9 +15796,9 @@ PyNumberMethods igraphmodule_Graph_as_number = {
   (unaryfunc) igraphmodule_Graph_complementer_op, /*nb_invert */
   0,                            /*nb_lshift */
   0,                            /*nb_rshift */
-  (binaryfunc) igraphmodule_Graph_intersection, /*nb_and */
+  0,                            /*nb_and */
   0,                            /*nb_xor */
-  (binaryfunc) igraphmodule_Graph_union,  /*nb_or */
+  0,                            /*nb_or */
```

Comment:
This row used to add support for using the `|` operator in Python between two graph objects to call the graph union function. I'm not 100% sure whether it was a good idea, but in order to keep the API stable, we must keep on supporting it. I don't mind if it is not supported any more by the low level `_GraphBase` object (which was never part of the public API), but we should re-add support for the `|` operator (in pure Python if that's easier) in the `Graph` object if we remove it from here.

Reply:
As above for the &